packages/cells/src/model.ts

```diff
@@ -492,6 +493,36 @@ export class CodeCellModel extends CellModel implements ICodeCellModel {
       this.onGenericChange,
       this
     );
+
+    // We keep `collapsed` and `jupyter.outputs_hidden` metadata in sync, since
+    // they are redundant in nbformat 4.4. See
+    // https://github.com/jupyter/nbformat/issues/137
+    this.metadata.changed.connect(
+      Private.collapseChanged,
+      this
+    );
+
+    // Sync `collapsed` and `jupyter.outputs_hidden` for the first time, giving
+    // preference to `collapsed`.
+    if (this.metadata.has('collapsed')) {
+      let collapsed = this.metadata.get('collapsed');
+      Private.collapseChanged(this.metadata, {
+        type: 'change',
+        key: 'collapsed',
+        oldValue: collapsed,
```

Comment:
Should the `oldValue` be different?

Reply:
I think technically it is correct, and it doesn't matter anyway in this particular invocation.